app/views/file_manager/index.html.erb

```diff
@@ -0,0 +1,92 @@
+<% end %>
+
+<% content_for :stylesheets do %>
+  <%= stylesheet_link_tag "file_manager" %>
+<% end %>
+
+<div class="div-row">
+  <%= form_tag({}, multipart: true) do %>
+    <div class="div-row-cols">
+      <span class="btn button-file">
+        <i class="material-icons left">cloud_upload</i>
+        Upload Files
+        <input type="file" name="files" id="fileInput" multiple
+               onchange="uploadAllFiles('<%= @directory[1][:relative] %>')">
+      </span>
+    </div>
+  <% end %>
+  <span class="btn button-file">
+    <i class="material-icons left">folder</i>
+    Create Folder
+    <input type="submit" id="create-folder"
+           onclick="createFolder('<%= @directory[1][:relative] %>')" class="btn primary">
+  </span>
+  <span class="btn button-file">
+    <i class="material-icons left">download</i>
+    Download Selected
+    <input type="submit" id="download-selected" class="btn primary" onclick="handleDownloadClick()">
+  </span>
+  <span class="btn button-file">
+    <i class="material-icons left">delete</i>
+    Delete Selected
+    <input type="submit" id="delete-selected" class="btn primary" onclick="handleDeleteSelected()">
+  </span>
+</div>
+<div class="table-scroll">
+  <table class="table">
+    <thead>
+    <tr>
+      <th>
+        <%= check_box_tag "check-all", "check-all", false, { class: "check-all" } %>
+      </th>
+      <th>Filename</th>
+      <th>Bytes</th>
+      <th>Date</th>
+      <th>Rename</th>
+      <th>Delete</th>
+    </tr>
+    </thead>
+    <tbody>
+    <% @directory.each do |entry| %>
+      <% if entry[:instructor] && entry[:entry] != './' %>
+        <tr class="table-row-center">
+          <td>
+            <% if entry[:entry] != '../' %>
+              <%= check_box_tag 'selected_files[]', entry[:relative], false, { class: "check", html: { data: { path: entry[:relative] } } } %>
+            <% end %>
+          <td>
+            <%= link_to entry[:relative].to_s do %>
+              <i class='material-icons left middle'><%= entry[:type] == :file ? 'article' : 'folder' %></i>
+              <span><%= entry[:entry] %></span>
+            <% end %>
+          </td>
+          <td>
+            <%= entry[:size] %>
+          </td>
+          <td>
+            <%= entry[:date] %>
+          </td>
+          <td class="table-row-icon">
+            <% if entry[:entry] != './' && entry[:entry] != '../' %>
+              <%= link_to "#", id: "rename-file", onclick: "rename('#{entry[:relative]}')" do %>
+                <i class='material-icons left align-middle'>drive_file_rename_outline</i>
+                Rename
+              <% end %>
+            <% end %>
+          </td>
+          <td>
+            <% if entry[:entry] != './' && entry[:entry] != '../' %>
+              <%= link_to "#", id: "delete-file", data: { confirm: "Are you sure you want to delete `#{entry[:absolute]}`?" }, onclick: "deleteSelected('#{entry[:relative]}')" do %>
```

Comment:
Confirmation doesn't seem to work properly, I don't think onclick waits for the confirm